sdk/tests/conformance/extensions/oes-texture-float.html

```diff
@@ -157,7 +157,8 @@
     }
     gl.texImage2D(gl.TEXTURE_2D, 0, format, width, height, 0, format, gl.FLOAT, data);
     if (expectFailure) {
-        wtu.glErrorShouldBe(gl, gl.INVALID_ENUM, "floating-point texture allocation must be disallowed if OES_texture_float isn't enabled");
+        wtu.glErrorShouldBe(gl, [gl.INVALID_ENUM, gl.INVALID_OPERATION],
```

Comment:
As above, the only error that should be possible from the texImage2D call above is INVALID_ENUM, since gl.FLOAT isn't supported because the OES_texture_float extension hasn't been enabled.